src/draw/AutomaticLayout.js

```diff
@@ -0,0 +1,406 @@
+import ELK from 'elkjs';
+
+/**
+ * A component data for automatic layout.
+ * Used within {@link AutomaticLayout}.
+ *
+ * It has tree-related fields : parent, children, depth,
+ * in addition to the fields from {@link Component} in {@link raw}.
+ * @typedef {object} NodeData
+ * @property {NodeData|null} parent - The node's parent
+ * @property {Component|null} raw The underlying component
+ * @property {NodeData[]} children This node's children
+ * @property {number} depth distance from the root (root's depth is 0)
+ */
+
+/**
+ * Parameters for the ELK automatic layout system.
+ * To be used within {@link AutomaticLayout}.
+ * @typedef {object} ElkParams
+ * @property {boolean} interactive whether we use the interactive mode
+ * @property {boolean} separateConnectedComponents see ELK documentation
+ * @property {string} cycleBreaking see ELK documentation
+ * @property {string} layering see ELK documentation
+ * @property {string} crossingMinimization see ELK documentation
+ * @property {string} nodePlacement see ELK documentation
+ * @property {string} interactiveReferencePoint see ELK documentation
+ * @property {string} direction see ELK documentation
+ */
+
+// eslint-disable-next-line jsdoc/require-property
+/**
+ * An ELK port.
+ * (external)
+ * @typedef {object} ElkPort
+ * @see elk-api.d.ts
+ */
+
+// eslint-disable-next-line jsdoc/require-property
+/**
+ * An ELK edge.
```

Comment:
see section is for link, so complete it with correct elk documentation link for all @see.
Or not sure that this typedef is relevant because. So update its documentation or remove it.

Reply:
removed